scripts/toolchain/Dockerfile

```diff
@@ -30,6 +30,8 @@ RUN apt-get update \
  && rm -rf /var/lib/apt/lists/* \
  && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false
 
+# Hoprd Docker image copies files from .cargo
+COPY .cargo .cargo
```

Comment:
?? Copies self into self?

Reply:
Copying *directories* in Docker requires specifying the destination folder. `COPY .cargo .cargo` means:
Take the `.cargo` directory from the monorepo and copy all its contents into a `.cargo` directory in the Docker image

Reply:
Ah ok, mistaken it for `RUN cp` :)